sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,29 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
-    packages = ('hbn-repo',)
+    packages = ('hbn-repo')
```

Comment:
we need a comma (`,`) here because packages has to be list :(

Reply:
done